<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebugPanel</class>
 <widget class="QDockWidget" name="DebugPanel">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Csound Debugger</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item row="1" column="1" colspan="4">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Current Stack</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="7">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QToolButton" name="toolButton">
        <property name="text">
         <string>Show</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Breakpoints</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="newBreakpointToolButton">
        <property name="text">
         <string>New</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/themes/boring/gtk-new.png</normaloff>:/themes/boring/gtk-new.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="deleteBreakpointToolButton">
        <property name="text">
         <string>Delete</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/themes/boring/gtk-close.png</normaloff>:/themes/boring/gtk-close.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="0" column="1" colspan="7">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QToolButton" name="runButton">
        <property name="text">
         <string>Run</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/themes/boring/gtk-media-play-ltr.png</normaloff>
          <normalon>:/themes/boring/gtk-media-stop.png</normalon>:/themes/boring/gtk-media-play-ltr.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="pauseButton">
        <property name="text">
         <string>Pause</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/themes/boring/gtk-media-pause.png</normaloff>:/themes/boring/gtk-media-pause.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="continueButton">
        <property name="text">
         <string>Continue</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/themes/boring/gtk-reload.png</normaloff>:/themes/boring/gtk-reload.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="nextButton">
        <property name="text">
         <string>Next</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="statusLabel">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 255, 127);
background-color: rgb(0, 0, 0);
font: 10pt &quot;Courier New&quot;;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="text">
         <string>Debugger stopped</string>
        </property>
        <property name="margin">
         <number>8</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="3" column="1" colspan="7">
     <widget class="QTableWidget" name="stackTableWidget">
      <attribute name="horizontalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>1</number>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Variable</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Value</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Type</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="5" column="1" colspan="7">
     <widget class="QTableWidget" name="breakpointTableWidget">
      <property name="cornerButtonEnabled">
       <bool>false</bool>
      </property>
      <property name="columnCount">
       <number>2</number>
      </property>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>1</number>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Type</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Line/Instrument</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>pause()</slot>
  <slot>continue()</slot>
  <slot>next()</slot>
 </slots>
</ui>
